daemon/errors.go

```diff
@@ -6,9 +6,13 @@ import (
 )
 
 var (
-	ErrUnexpected        = errors.NewKind("unexpected error")
-	ErrMissingDriver     = errors.NewKind("missing driver for language %q")
-	ErrRuntime           = errors.NewKind("runtime failure")
-	ErrAlreadyInstalled  = protocol.ErrAlreadyInstalled
+	ErrUnexpected       = errors.NewKind("unexpected error")
```

Comment:
Please document all these errors (so they have something explanatory in godoc).

Reply:
Have documented the only one error so far, that was added in this PR.

The rest, to the best of my knowlege, is added in e3f38d8

Would prefer to do further godoc updates for the code that was not changed in PR in a separate effort like #211